src/app/customerlist/page.js

```diff
@@ -1,49 +1,14 @@
 "use client";
 
-import { Box, Grid, useMediaQuery, useTheme } from "@mui/material";
-import React, { useState } from "react";
+import { Box, Grid, Typography, useMediaQuery, useTheme } from "@mui/material";
```

Comment:
![image](https://github.com/user-attachments/assets/ec2f699b-1150-496b-8674-f33b3f9d516a)
Gird is deprecated to use . Please double check using Grid in importing  pre-component lib.